app/javascript/controllers/nested_form_controller.js

```diff
@@ -1,7 +1,10 @@
 import { Controller } from "stimulus"
 
 export default class extends Controller {
-  static targets = ["add_item", "template"]
+  // Declare targets for this controller and its children because adding to
+  // inherited targets in children causes Stimulus controller errors
+  // ("ordered_nested_form_controller_default is not defined")
+  static targets = ["add_item", "template", "upButton", "downButton"]
```

Comment:
Will it error on target not found when using `data-controller="nested-form"`?

Reply:
@jcoyne I'm not sure I follow, but the build is already broken so I'm not exactly rushing to get this in. What all changes are you suggesting I might want to try?

Reply:
I'm not suggesting any change in particular. I was just wondering if we've tested both the `action-controller="nested-form"` and `action-controller="ordered-nested-form"`